WordPress/gutenberg
# Changed labels to consistently use Patterns in favor of Block patterns

## What?

Consistently use "Pattern" rather than "Block pattern" the UI labels 

## Why?

Fixes #49617

## How?

Searched repo for __\(.*patterns and replaced the 4 instances where "block patterns" was used

## Testing Instructions

1. Open a post or page. -->
2. Insert a block. -->
3. choose "browse all -->
4. search for hero, cta, etc
5. Inspect the page and search for "block patterns" in the document; the visually hidden label for the patterns reads patterns.
6. Insert a pattern. The success notice will say "Pattern %s inserted" rather than "Block pattern %s inserted"

### Testing Instructions for Keyboard


## Screenshots or screencast
